README.md

```diff
@@ -111,62 +151,39 @@ https://hub.docker.com/r/dsnp/hardhat/tags
 
 Exposed Port: 8545
 
-#### Accounts 
+## Test Accounts 
 
-These are setup in the [hardhat.config.ts](https://hardhat.org/config/#hardhat-network) file.
-If you need different accounts, please build a different image.
+These are set up in the [hardhat.config.ts](https://hardhat.org/config/#hardhat-network) file.
+If you need different accounts, please build a different image with a different mnemonic.  Set the chain ID to your choice.
 
 ```
-MNEMONIC="test test test test test test test test test test test junk"
+MNEMONIC="this is some other mnemonic test test test"
```

Comment:
We should likely at least have a valid mnemonic here or perhaps a link to generate one?

Reply:
I don't think that's a good idea as it could lead a lot of people to use it and accidentally forget they're using it, and lose funds -- and blame us.  Instead I think we could just make it "your valid mnemonic here" or something, perhaps point them to a generator.